Script/CandySpawner.gd

```diff
@@ -16,6 +16,7 @@ extends Node2D
 var _texture_index := 0
 
 const FALLBACK_TEXTURE : Texture2D = preload("res://Image/Candy.png")
+var _MAX_SIZE = FALLBACK_TEXTURE.get_size().x
```

Comment:
We might want to do the same for goobers, here, but we can see how things go in practice.